Test/Core/Components/RigidBodyTest.cs

```diff
@@ -1,21 +1,54 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.IO;
-using System.Reflection;
-
-using Duality;
-using Duality.Cloning;
+﻿using System.Linq;
+using System.Threading;
 using Duality.Components;
 using Duality.Components.Physics;
-
+using Duality.Resources;
 using NUnit.Framework;
 
 namespace Duality.Tests.Components
 {
 	[TestFixture]
 	public class RigidBodyTest
 	{
+		[Test] public void FallingBallOnPlatform()
+		{
+			var scene = new Scene();
+			Scene.SwitchTo(scene);
+
+			//Create the ball
+			var ball = new GameObject("Ball");
+			var ballTransform = ball.AddComponent<Transform>();
+			var ballBody = ball.AddComponent<RigidBody>();
+			var listener = ball.AddComponent<CollisionListener>();
+			bool onCollisionBeginCalled = false, onCollisionEndCalled = false, onCollisionSolveCalled = false;
+			listener.CollisionBegin += (sender, args) => onCollisionBeginCalled = true;
+			listener.CollisionEnd += (sender, args) => onCollisionEndCalled = true;
+			listener.CollisionSolve += (sender, args) => onCollisionSolveCalled = true;
+			ballBody.AddShape(new CircleShapeInfo(1, new Vector2(0, 0), 1));
+			scene.AddObject(ball);
+
+			//Create the platform
+			var platform = new GameObject("Platform");
+			var platformTrans = platform.AddComponent<Transform>();
+			var platformBody = platform.AddComponent<RigidBody>();
+			platformBody.AddShape(new ChainShapeInfo(new[] { new Vector2(-1, 0), new Vector2(1, 0), }));
+			platformBody.BodyType = BodyType.Static;
+			scene.AddObject(platform);
+			platformTrans.Pos = new Vector3(0, 5, 0);
+
+			for (int i = 0; i < 6; i++)
+			{
+				Thread.Sleep(16); //Simulate ~60 fps.
```

Comment:
Don't sleep! Instead, we can modify the Duality core to expose the same fixed-step time simulation that we get in the editor. Take a look at `DualityApp` around [here](https://github.com/AdamsLair/duality/blob/f72221bc75e27a2450d96915adb3513c5df94284/Source/Core/Duality/DualityApp.cs#L550). The `EditorUpdate` method has a `forceFixedStep` parameter. Add it to `Update` as well and use it in the test.

To make this not a breaking change, introduce a parameterless `Update` overload to `DualityApp` that simply calls the modified one with the new parameter being false.